labelbox/schema/identifiable.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC
+from enum import Enum
+from typing import List, Union
+from typing_extensions import TypeAlias
```

Comment:
TypeAlias seems to be available since 3.10

Reply:
yes, but I am using `typing_extensions` that back-ported it. However `TypeAlias` is not strictly necessary, it is just for improved typing. Should I remove it?